tsconfig.json

```
@@ -13,11 +13,13 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "verbatimModuleSyntax": true,
```

```suggestion
    "verbatimModuleSyntax": true,
    "noPropertyAccessFromIndexSignature": true,
    "noUncheckedIndexedAccess": true,
    "noFallthroughCasesInSwitch": true,
    "noImplicitOverride": true,
```
quelques petites règles ts que je trouve interessante a avoir : 
- https://www.typescriptlang.org/tsconfig/#noPropertyAccessFromIndexSignature
- https://www.typescriptlang.org/tsconfig/#noUncheckedIndexedAccess
- https://www.typescriptlang.org/tsconfig/#noFallthroughCasesInSwitch
- https://www.typescriptlang.org/tsconfig/#noImplicitOverride

---

C'est ajouté pour les 3 premières !
Indeed pas trop l'intention de faire des classes pour l'instant donc la quatrième ne devrait pas m'être utile !
Thanks 🙏